### ISSUE TITLE
questions about batch norm layer

### ISSUE THREAD

**Issue by StephenZhao1** [meteorshowers/RCF-pytorch#59]
I have read some source code about many edge detection network, all of the network are without BN layer, why was that?
